### PR TITLE
Picker example for the webDemo

### PR DESCRIPTION
**Pull request by adids1221**
## Description
Picker example for the web demo.

## Changelog
Picker example for the web demo.

### CODE REVIEW

File: webDemo/src/App.tsx [hunk 1]
```diff
@@ -374,15 +374,15 @@ function App() {
   return (
     <View flex>
       <ScrollView>
-        <View style={styles.header}>
-          <Text style={styles.title}>Welcome to react-native-ui-lib for Web</Text>
+        <View padding-page>
+          <Text text40H center marginV-s1>Welcome to wix-react-native-ui-lib for Web</Text>
```
Comment (lidord-wix) [inline webDemo/src/App.tsx:378]: The public library name doesn't include `wix` 
```suggestion
          <Text text40H center marginV-s1>Welcome to react-native-ui-lib for Web</Text>
```

File: webDemo/src/App.tsx [hunk 2]
```diff
@@ -399,30 +399,10 @@ function App() {
...
-    fontWeight: 'bold',
-    fontSize: '1rem',
-    marginVertical: 4,
-    textAlign: 'center'
-  },
-  title: {
-    fontWeight: 'bold',
-    fontSize: '1.5rem',
-    marginVertical: '1em',
-    textAlign: 'center'
```
Comment (lidord-wix) [inline webDemo/src/App.tsx:425]: Seems like the typography modifiers don't work well on the web (mainly the font-weight I guess).
I suggest to returned it back to use the stylesheet, unless you want to change the design :)

File: webDemo/src/examples/Picker.tsx [hunk 3]
```diff
@@ -10,28 +9,61 @@ const options = [
...
-      searchPlaceholder={'Search a language'}
-      searchStyle={{color: Colors.blue30, placeholderTextColor: Colors.grey50}}
-      migrateTextField
-    >
-      {options.map(option => (
-        <Picker.Item key={option.value} value={option} label={''} disabled={option.disabled}/>
-      ))}
-    </Picker>
+    <View marginT-s2>
+      <Text text80BO marginT-s1 center>Single Value Picker</Text>
```
Comment (lidord-wix) [inline webDemo/src/examples/Picker.tsx:27]: Can we add an example where the picker opened in a dialog?

**Review by lidord-wix — COMMENTED** [review]
looks good! I added some comments :)

File: webDemo/src/examples/Picker.tsx [hunk 4]
```diff
@@ -10,28 +10,110 @@ const options = [
...
+    const {visible, children, toggleModal, onDone} = modalProps;
+
+    return (
+      <Incubator.Dialog
+        visible={visible}
+        onDismiss={() => {
+          onDone();
+          toggleModal(false);
+        }}
+        width="100%"
```
Comment (lidord-wix) [inline webDemo/src/examples/Picker.tsx:42]: I'd decrease the width to make it look more like a dialog :)

**Review by lidord-wix — COMMENTED** [review]
See my comments :)

**Comment by lidord-wix** [comment]
@adids1221 
I also pushed a small fix here by mistake (I planned to push it to master) 
So please pull locally when working on this branch 🙏🏽

File: webDemo/src/App.tsx (hunk 2 above)
Comment (adids1221) [reply webDemo/src/App.tsx:425]: Returned to the stylesheet, in the redesign I'll use our modifiers.

**Review by lidord-wix — COMMENTED** [review]
Looks good.
What about the svg issue?

**Comment by adids1221** [comment]
> Looks good. What about the svg issue?

I've opened a ticket for this issue, I'll try to get to it next week.

**Review by lidord-wix — APPROVED** [review]
Approved, just resolve the conflict and merge :)

**Pull request merged**